contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -227,6 +227,7 @@ Additionally we can improve messages to comply with good practices:
   the subject would be the program itself ("Could not do B").
 * Use past tense ("Could not delete B" instead of "Cannot delete B")
 * When referring to an object, state what type of object it is. ("Deleted pod" instead of "Deleted")
+* Avoid usage of a period whenever possible, try to use a comma instead (see the example below)
```

Comment:
I think  we should recommend recommend log messages to be a single sentence.

Reply:
Yes, what should be the ideal way to migrate
 ```klog.Errorf("unable to determine dual-stack status. Falling back to single-stack %v", err)```?

Reply:
I somewhat rephrased your text, using single sentence imply not using period and preferring comma. In example you give we would use a complex sentence, constructed out of multiple simple sentences (clauses) connected by comma. Using complex sentence also implies that there should be direct dependence between them.

I'm not english native so I'm struggling a little with explanation, maybe we should ask someone from SIG-contributex or SIG-docs to help get the message accross.

Reply:
Okay, I'll reach out to SIG-docs in slack to get more suggestions on this.